ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -36,6 +36,17 @@ import { LoadingModalComponent } from '../modals/loading-modal.component';
 import { BulkCohortTagModalComponent } from '../tags/cohort-tag/bulk-cohort-tag-modal/bulk-cohort-tag-modal.component';
 import { CohortTagComponent } from '../tags/cohort-tag/cohort-tag.component';
 import { CohortTag } from '../tags/cohort-tag/cohort-tag.model';
+import {FieldSettingsModel, ValueModel} from '../STORE/models';
+
+interface IAdditionalValues {
```

Comment:
did you check for other studies with additional fields like Circadia and RGP? do we need to add the tab names in the frontend now?

Reply:
Do we need to check for studies? 
My code simply just maps new values to `additionalValuesJson` field.
If there is no `additionalValuesJson`, returns just null and the template won't be rendered, or if there is no value to map, simply just won't map. 
You can find my code from 809th line (`parseAdditionalValues` method)

Another part I didn't understand

Reply:
Don't know how they should look for the other studies :( If you find any issues, let me know and I will adjust.

Reply:
**Update:**
Removed `IAdditionalValues` interface, just to avoid confusion as in additionalValuesJson field might be different objects stored